### PR TITLE
Updated project documentation to use the useblocks theme

### PR DESCRIPTION
**Pull request by iSOLveIT**

### CODE REVIEW

File: docs/ub_theme/SUBTREE.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+The ub_theme folder is a reference to the [ub_theme project](https://github.com/useblocks/ub_theme) - do not edit.
+Instead, you can send your changes to the ub_theme project.
```
Comment (danwos) [inline docs/ub_theme/SUBTREE.md:2]: Can we add here the needed command to update the data in this folder via git-subtree?

File: pyproject.toml [hunk 2]
```diff
@@ -10,7 +10,7 @@ documentation = "https://sphinx-data-viewer.readthedocs.io/en/latest/"
 
 [tool.poetry.dependencies]
 python = "^3.6"
-sphinx = "^4"
+sphinx = "^5"
```
Comment (danwos) [inline pyproject.toml:13]: Is there a reason for throwing out Sphinx 4?

I think we should just write `>=4`

**Review by danwos — CHANGES_REQUESTED** [review]

File: docs/ub_theme/SUBTREE.md (hunk 1 above)
Comment (iSOLveIT) [reply docs/ub_theme/SUBTREE.md:2]: Yeah sure

File: docs/ub_theme/README.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+## Installation
+To install the files from this repository, you must have [Git](https://git-scm.com) installed.
+
+* Change directory to your preferred docs working directory
+  ```bash
+  cd docs
+  ```
+* Download the files from the repository:
+  ```bash
+  git clone https://github.com/useblocks/ub_theme.git
```
[the remark this answers is not in the document]
Comment (iSOLveIT) [reply docs/ub_theme/README.md:19]: Yeah I have to update that as well

File: pyproject.toml (hunk 2 above)
Comment (iSOLveIT) [reply pyproject.toml:13]: There's no reason though. I haven't made any changes to the pyproject.toml for the extension

File: docs/ub_theme/README.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+  ```
+
+> **NOTE:** 
+> 
+> You can update the _ub_theme_ repository by using:
+> ```bash
+> git subtree pull --prefix docs/ub_theme/ https://github.com/useblocks/ub_theme.git main --squash
+> ```
+
+## Configuration
```
Comment (danwos) [inline docs/ub_theme/README.md:26]: Maybe a small chapter "update" would be also good, which explains the command to get the latest changes from ubtheme repo

**Review by danwos — CHANGES_REQUESTED** [review]

File: docs/ub_theme/SUBTREE.md (hunk 1 above)
Comment (danwos) [reply docs/ub_theme/SUBTREE.md:2]: Mmh, still can't see the command in this file to update the subtree.

Comment (iSOLveIT) [reply docs/ub_theme/SUBTREE.md:2]: Oh okay. I added the command to the README.md file instead.

**Review by danwos — APPROVED** [review]

**Pull request merged**